### PR TITLE
chore: fix status mappings after API updates

### LINKED ISSUE #1116: Fix status mappings after API updates

### PR DESCRIPTION
**Pull request by MadalinaRaicu**
# Related issues 🔗

Closes #1116 

# Description ℹ️

Fix status mappings after API updates

# Demo 📺
image

### CODE REVIEW

**Review by edd — APPROVED** [review]
:+1: I'm going to go and test it manually as a backup, but this looks ~really tedious~ good to merge!

**Review by sam-keen — APPROVED** [review]

**Comment by edd** [comment]
:+1: Manually verified

image

image

File: libs/market-list/src/lib/components/select-market-columns.tsx [hunk 1]
```diff
@@ -217,11 +231,11 @@ export const columns = (market: any, onSelect: (id: string) => void) => {
       onlyOnDetailed: false,
     },
     {
-      value: market.lastPrice ? (
+      value: market.data?.markPrice ? (
         <PriceCell
-          value={new BigNumber(market.lastPrice).toNumber()}
+          value={new BigNumber(market.data?.markPrice).toNumber()}
```
Comment (mattrussell36) [inline libs/market-list/src/lib/components/select-market-columns.tsx:235]: question (non blocking): Why create big number here and then convert to number anyway? Will `Number(market.data?.markPrice ?? 0)` suffice?

Comment (mattrussell36) [reply libs/market-list/src/lib/components/select-market-columns.tsx:235]: Same elsewhere in this file..

**Review by mattrussell36 — CHANGES_REQUESTED** [review]
I have a question in here but also please fix the lint errors

**Review by mattrussell36 — APPROVED** [review]

**Pull request merged**